controllers/utils_test.go

```diff
@@ -168,6 +169,36 @@ func getTestPolicyServerService(policyServerName string) (*corev1.Service, error
 	return &service, nil
 }
 
+func getTestPolicyServerDeployment(policyServerName string) (*appsv1.Deployment, error) {
+	policyServer := policiesv1.PolicyServer{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: policyServerName,
+		},
+	}
+	deploymentName := policyServer.NameWithPrefix()
```

Comment:
I think we have bunched of places doing the same thing. Which signals me that this piece of code is a good candidate for a dedicated function. However, this is not a comment that should block merging this PR